install_config/configuring_vsphere.adoc

```diff
@@ -188,6 +193,11 @@ If the file does not exist, create it, and add the following:
 <8> (Optional) The vCenter VM folder path in which the node VMs are located. It can be set to an empty path(`working-dir = ""`), if Node VMs are located in the root VM folder.
 <9> (Optional) VM Instance UUID of the Node VM. It can be set to empty (`vm-uuid = ""`). If this is set to empty, this is retrieved from *_/sys/class/dmi/id/product_serial_* file on virtual machine (requires root access).
 
+[NOTE]
```

Comment:
I would consider changing this note to:

Remove special characters from the username and password fields. For example, user = "domain\\username".